autopush/logging.py

```diff
@@ -113,12 +113,17 @@ def __call__(self, event):
 
     def raven_log(self, event):
         f = event["log_failure"]
-        stack = extra = None
+        stack = None
+        extra = dict()
         tb = f.getTracebackObject()
         if not tb:
             # include the current stack for at least some context
             stack = list(iter_stack_frames())[4:]  # approx.
             extra = dict(no_failure_tb=True)
+        extra.update(log_format=event.get('log_format'),
+                     log_namespace=event.get('log_namespace'),
+                     why=event.get('why'),
+                     client_info=event.get('client_info'))
```

Comment:
Is this supposed to be getattr(event...) ? Below its attached as fail._client_info, does twisted change attributes on the fail into the event dict?